tests/transforms/test_experimental/test_transform_program.py

```diff
@@ -564,26 +588,6 @@ def transform_mul(tape: qml.tape.QuantumTape):
         assert len(new_batch) == 1
         assert new_batch[0] is tape0
 
-        assert fn.func is _apply_postprocessing_stack
```

Comment:
A lot of tests are removed here, can you tell more about that?

Reply:
They were removed because of my changes to `TransformProgram.__call__`. Because of the `postprocessing_fn` no longer being `functools.partial`, all of the tests failed because we can't inspect the function like we could before.